apps/block/templates/_sidebar.jade

```diff
@@ -6,7 +6,7 @@
       include info/_description
       include info/_connections
 
-      if user && user.has('name')
+      if user && user.has('username')
```

Comment:
Yeah, the `arena-passport` middleware injects a user into `res.locals` that is just an empty backbone model